lts/pkg/src/005-1-invalid-au/005-1-invalid-au.js

```diff
@@ -414,22 +414,25 @@ const saveLearnerPrefs = async (
                 }
             },
             {
-                // statement with actor objectType set to "Group"
+                // defined statement with actor objectType set to "Group"
                 reqId: "9.2.0.0-2",
+                type: "passed",
                 alter: (st) => {
                     st.actor.objectType = "Group";
                 }
             },
             {
-                // statement with actor objectType set to an invalid value
+                // defined statement with actor objectType set to an invalid value
```

Comment:
Here the actor itself is invalid, so Engine returns a http `400` on failure to deserialize the actor before even getting to cmi5 validations.

So the change to this being a defined statement isn't enough, we should also accept `400` (and indeed, I'm not sure we should accept `403`, because xAPI seems to demand `400`):

> 400 Bad Request - Indicates an error condition caused by an invalid or missing argument. The term "invalid arguments" includes malformed JSON or invalid Object structures.

And although the spec could be clearer here, where xAPI defines the "Actor" property it also indicates that the actor can be an agent, in which case the objectType is optional but must be "agent" if present, or a group, in which case the object type is required and must be "group"

https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#242-actor

Reply:
I've updated this and helpers.js to reflect this opinion.  Back to you!